lib/diagnostic.js

```diff
@@ -6,23 +6,36 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (paragraph) {
+  let words = paragraph.split(/\s+/);
+  let normalizedWords = [];
+  for (let i = 0; i < words.length; i++) {
+    let word = word[i].replace(/\W+/ '');
+    normalizedWords = toUpperCase;
```

Comment:
What is `normalizedWords` on line 14? It's an array. `toUpperCase` only works on words. How do we save things into arrays?